### PR TITLE
Minor fixes for Cpp code samples [API-598]

### PR DESCRIPTION
**Pull request by hakanaktas0**
Some unnecessary parts are removed and sample code is altered to make its output similar to the given output.

### CODE REVIEW

File: cpp-unisocket/main.cpp [hunk 1]
```diff
@@ -1,27 +1,20 @@
...
     while (true) {
         int random_key = rand() % 100000;
         try {
             map->put("key-" + std::to_string(random_key), "value-" + std::to_string(random_key));
-            if (random_key % 100 == 0) {
-                std::cout << "Current map size: " + std::to_string(map->size().get()) << std::endl;
-                std::this_thread::sleep_for(std::chrono::seconds(1));
-            }
+            std::cout << "Current map size: " + std::to_string(map->size().get()) << std::endl;
+            std::this_thread::sleep_for(std::chrono::seconds(1));
```
Comment (ihsandemir) [inline cpp-unisocket/main.cpp:17]: Most client examples, except dotnet, do not sleep for every put, what is the reason for this change?

Comment (hakanaktas0) [reply cpp-unisocket/main.cpp:17]: Looks better on the console, without the sleep it populates the map too fast and text becomes unreadable on console. It is a cosmetic change, duration of the sleep may be shortened, but I think it is better this way.

**Review by ihsandemir — APPROVED** [review]
